### ISSUE TITLE
Create question employee dropdown is not working

### ISSUE THREAD

**Issue by albertcastaned** [wizeline/wize-q-remix#150]

**Comment by Carlos-Reyna** [comment]
Hello, can you please provide more context on this bug? I see its working on my local 👀

**Comment by albertcastaned** [comment]
Sorry didn't finish writing the issue. Testing in https://questions-develop.wizeline.com/questions/new I get the issue from screenshot demonstrated. I havent digged into the database, hopefully there is not duplicate entries of the employee. Getting the same for other deparments.

**Comment by Carlos-Reyna** [comment]
Oh, while I was developing this feature i came across that the database seed has a ton of duplicates on the EmployeesDepartments table, I don't know if the same could be happening on Testing but on this bug we can add a feature to not retrieve duplicates by department

**Comment by albertcastaned** [comment]
Yep. Looks like there are multiple duplicates in QA Database. Probably how seeeding worked in previous application. Will see if the production database looks the same. On the meantime, I think the safest option would be to add a unique filter to the Prisma query to avoid duplicate @Carlos-Reyna

**Comment by GeorginaFraire** [comment]
@Carlos-Reyna  you can use [distinct](https://www.prisma.io/docs/concepts/components/prisma-client/aggregation-grouping-summarizing#select-distinct) to avoid this kind of things 
```const relations = await db.EmployeesDepartments.findMany({
    select: {
      email: true,
      department_id: true,
      employee_id: true,
      users: {
        select: {
          employee_id: true,
          full_name: true,
          email: true,
          is_admin: true,
          profile_picture: true,
          job_title: true,
        },
      },
    },
    where: {
      department_id: idValue,
    },
    distinct: ['email', 'department_id', 'employee_id'],
  });```

**Comment by albertcastaned** [comment]
@GeorginaFraire I believe its already done haha https://github.com/wizeline/wize-q-remix/pull/153
